New README for FastAI (#14)
FluxML/FastAI.jl: README.md

![Logo](https://raw.githubusercontent.com/opus111/FastAI.jl/master/fastai-julia-logo.png)

> FluxAI.jl is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), and is a repository of best practices for Deep Learning with Flux.  Its goal is enable the development of state-of-the-art models, while freeing the developer from having to implement most of the components.  FastAI allows the design, training and delivery of models that compete with the best in class, using a few lines of code.  

Reviewer: ```suggestion
FastAI.jl is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), and is a repository of best practices for Deep Learning with Flux. Its goal is to enable creating state-of-the-art models, while freeing the developer from having to implement most of the sub-components. FastAI allows the design, training, and delivery of models that compete with the best in class, using a few lines of code.
```

FastAI.jl contains thorough documentation, examples and tutorials, but does not contain the source of the core components.  It is an umbrella package combining the functionality specialized packages.  These packages include:

> - Flux.jl: 100% pure-Julia Deep Learning stack. Provides lightweight abstractions on top of Julia's native GPU and AD support.

Reviewer: ```suggestion
- Flux.jl: 100% pure-Julia Deep Learning stack. Provides lightweight abstractions on top of Julia's core GPU and AD support.
```

> - FluxTraining.jl: Enables eaily customized training loops, a large library of useful metrics, and many useful utilities

Reviewer: ```suggestion
- FluxTraining.jl: Easily customized training loops, a large library of useful metrics, and many useful utilities (such as logging)
```

> - FluxONNX.jl: Support for reading and writing models in the ONNX file format.  ONNX enables Transfer Learning by enabling the import of very large pretrained models.

Reviewer: Should we leave this as a comment for now until we have an official ONNX package?

Reviewer: Any comments on this?

Reviewer 2: Yeah, I agree

- MLDataPattern.jl: Utility package for subsetting, partitioning, iterating, and resampling of Machine Learning datasets.

- MLDataSets.jl: A community effort to provide a common interface for accessing common Machine Learning (ML) datasets.

- DataAugmentation.jl: Utilities for augmenting image data

- Metalhead.jl: Computer vision models for Flux 

- Transformers.jl: Implementation of NLP transformer-based models

> *Note*: this is a package in-development. It is not ready for use, and one should expect major breaking changes for the foreseeable future. However, we are very interested in meeting the desires of the community, so all comments and contributions are welcome via PRs/issues.

Reviewer: ```suggestion
*Note*: this is a package in-development. One should expect major breaking changes for the foreseeable future. But we are very interested in meeting the desires of the community, so all comments and contributions are welcome via PRs/issues.
```
